cfme/physical/physical_storage.py

```diff
@@ -0,0 +1,231 @@
+# -*- coding: utf-8 -*-
+"""A model of an Infrastructure PhysicalStorage in CFME."""
+import attr
+from navmazing import NavigateToAttribute
+from cached_property import cached_property
+
+from cfme.common import PolicyProfileAssignable, Taggable
+from cfme.common.physical_storage_views import (
+    PhysicalStorageDetailsView,
+    PhysicalStoragesView
+)
+from cfme.exceptions import ItemNotFound
+from cfme.modeling.base import BaseEntity, BaseCollection
+from cfme.utils.appliance.implementations.ui import CFMENavigateStep, navigate_to, navigator
+from cfme.utils.log import logger
+from cfme.utils.pretty import Pretty
+from cfme.utils.providers import get_crud_by_name
+from cfme.utils.update import Updateable
+from cfme.utils.wait import wait_for
+
+
+@attr.s
+class PhysicalStorage(BaseEntity, Updateable, Pretty, PolicyProfileAssignable, Taggable):
+    """Model of an Physical Storage in cfme.
+
+    Args:
+        name: Name of the physical Storage.
+
+    Usage:
+
+        mystorage = PhysicalStorage(name='sample_Storage')
+        mystorage.create()
+
+    """
+    pretty_attrs = ['name']
+
+    name = attr.ib()
+    provider = attr.ib(default=None)
+
+    INVENTORY_TO_MATCH = ['product_name', 'serial_number', 'description',
+                          'health_state', 'drive_bays', 'enclosure_count']
+
+    def load_details(self, refresh=False):
```

Comment:
Required:
No need to add this method. It should cover in test only not in model